getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -71,15 +71,15 @@ Project setup and workflow
 --------------------------
 
 When you create the first C# script, Godot initializes the C# project files for your Godot project.
-This includes generating a C# solution (``.sln``) and project (``.csproj``) as well as some utility files and folders (``.mono``, sometimes ``Properties``).
+This includes generating a C# solution (``.sln``) and project (``.csproj``) files, as well as some utility files and folders (``.mono``, sometimes ``Properties``).
```

Comment:
"a" should be removed.
Alternatively, one often names the `.sln` file "a C# solution", so it could be "a C# solution () and a project () file".

Reply:
> "a" should be removed.
> Alternatively, one often names the `.sln` file "a C# solution", so it could be "a C# solution () and a project () file".

Yup, good suggestion! :smile: 
> "a C# solution () and a project () file"

it is!